### ISSUE TITLE
Suggestion:Redirect to Menu on Menu Button Press in Bottom Navigation Bar when the user is in the menu section of the app.

### ISSUE THREAD

**Issue by Ashurajput02** [thunderbiscuit/padawan-wallet#353]
Hi @artfuldev @Prakhar-Agarwal-byte @geigerzaehler242 .
Whenever the user hits the menu button, they come across some options to navigate. For instance, if they open the "Recovery phase," the "menu" button in the bottom navigation bar becomes inactive. I want to suggest that the user should be redirected to the menu itself as soon as they press the 'menu' button again. They should not need to press the back button present at the top left corner.

I would be happy to resolve this minor but important issue.

https://github.com/thunderbiscuit/padawan-wallet/assets/119595975/e8889ad1-269e-4a9d-8f8b-dd9c8abf747b

See even on pressing the "Menu" button the user is not redirected back . He/She is redirected only after clicking the back button present on the top left.

**Comment by thunderbiscuit** [comment]
Agreed this would be a nice little fix! Feel free to work on this @Ashurajput02.
